.github/issue-labler.yml

```diff
@@ -0,0 +1,3 @@
+Status: 
+  Triage:
+    - '(Describe the bug)'
```

Comment:
Where did you get this regex from?
We want all opened issues to gain the label.
It should be all opened issues, per https://github.com/github/issue-labeler?tab=readme-ov-file#label-all-issues

Reply:
>  It should be all opened issues, per https://github.com/github/issue-labeler?tab=readme-ov-file#label-all-issues

You were right